client/src/browse/BrowsePage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from "react";
+import styled from "styled-components";
+
+import Map from "./maps/Map.js";
+import ItemList from "./list/ItemList.js";
+import FilterBar from "./filterBar/FilterBar.js";
+
+const BrowsePage = (props) => {
+  const [lat, setLat] = useState(null);
+  const [lng, setLng] = useState(null);
+  const [itemData, setItemData] = useState([]);
+  const [filteredItems, setFilteredItems] = useState([]);
+  const [selectedItem, setSelectedItem] = useState(null);
+
+  useEffect(() => {
+    if (navigator.geolocation) {
+      navigator.geolocation.watchPosition(function (position) {
+        setLat(position.coords.latitude);
+        setLng(position.coords.longitude);
+      });
+    }
+  }, []);
+
+  useEffect(() => {
+    fetch("http://localhost:3000/local")
+      .then((res) => res.json())
+      .then((res) => {
+        const temp = [];
+        for (let i = 0; i < res.length; i += 100) {
+          temp.push(res[i]);
+        }
+        setItemData(temp);
+        setFilteredItems(temp);
+      });
+  }, []);
+
```

Comment:
@MarkAlperin I have some api fetch functions built out in the api folder. We should get together and talk about fetching your data. 

Something like this

`function get(endpoint, queries) {
  return runFetch(...buildGetOptions(endpoint, queries))
}

function getData(path, queries) {
  return get('/data' + path, queries)
}

function getDonations(queries) {
  return getData('/local/donations', queries)
}`

I also have the same for getting user location.

Reply:
Removed old fetch request, will run newer query or get data from Context in "integration" PR